rustfmt.toml

```diff
@@ -0,0 +1,11 @@
+edition = "2018"
```

Comment:
📣 Please introduce formatting rule in a dedicated PR where you also add a check in the CI to enforce those rules.

Reply:
I think there already is a check in CI, I introduced this because it was failing the check

Reply:
If your PR fails the format check then just run `cargo fmt` locally, no need to introduce `rustfmt.toml` or use nightly.